modules/react-native-status/desktop/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ ExternalProject_Add(StatusGo_ep
   PREFIX ${StatusGo_PREFIX}
   SOURCE_DIR ${StatusGo_SOURCE_DIR}
   GIT_REPOSITORY https://github.com/status-im/status-go.git
-  GIT_TAG v0.11.0
+  GIT_TAG develop-g19b53030
```

Comment:
why is it develop again? @adambabik

Reply:
As far as I know, Status Desktop builds status-go from the source. It means that it should be a git tag or git commit SHA. In this case, `19b53030`.

`develop-g19b53030` is the name of the artifact build for ObjC or Android.

@flexsurfer because there was no release `v0.11.1` yet. I will do that asap.

Reply:
https://github.com/status-im/status-go/releases/tag/v0.12.0

Reply:
@adambabik, looks like cmake can understand `develop-g19b53030` as well as sha.

Reply:
I have just checked that and indeed, git can handle this just fine.